CONTRIBUTING.md

```diff
@@ -181,10 +181,14 @@ Beware that some dependencies cannot be upgraded because of issues or
 incompatibilities, while others must be upgraded by following a specific
 process. This section documents such cases:
 
-- The major version number of `@types/node` must match the version of Node
-  specified in the `engine` field of `package.json`.
 - `d3-array` (and therefore `@types/d3-array`) cannot be upgraded to v3.x due to
   a [bug in Babel](https://github.com/babel/babel/issues/11038).
+- Updating `@testing-library/react` to v12 breaks a lot of tests, probably
```

Comment:
"probably" ? :smile:

Reply:
It was a bit scary, I did not investigate further. 🙄